app/src/main/res/layout/user_dictionary_add_word_fullscreen.xml

```diff
@@ -45,31 +47,51 @@
             android:id="@+id/user_dictionary_add_shortcut"
             android:layout_width="wrap_content"
             android:layout_gravity="fill_horizontal|center_vertical"
-            android:layout_marginBottom="8dip"
-            android:layout_marginStart="8dip"
-            android:layout_marginTop="8dip"
+            android:layout_marginBottom="8dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
             android:hint="@string/user_dict_settings_add_shortcut_hint"
             android:imeOptions="flagNoFullscreen"
             android:inputType="textNoSuggestions"
             android:maxLength="@integer/config_user_dictionary_max_word_length" />
 
+        <TextView
+            android:id="@+id/user_dictionary_add_frequency_label"
+            style="?android:attr/textAppearanceMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="start|center_vertical"
+            android:text="@string/user_dict_settings_add_frequency_name"
+            android:visibility="visible" />
+
+        <EditText
+            android:id="@+id/user_dictionary_add_frequency"
+            android:layout_width="wrap_content"
+            android:layout_gravity="fill_horizontal|center_vertical"
+            android:layout_marginBottom="8dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
+            android:hint="@string/user_dict_settings_add_frequency_hint"
```

Comment:
Maybe the hint could also be "250" to indicate the default frequency / weight

Reply:
Done with commit da2d29f.